datafusion-cli/src/command.rs

```diff
@@ -206,11 +243,31 @@ impl FromStr for Command {
                 Self::OutputFormat(Some(subcommand.to_string()))
             }
             ("pset", None) => Self::OutputFormat(None),
+            ("memory_profiling", sub) => {
```

Comment:
What do you think about supporting `\memory_profling` without an argument that works like a toggle to enable/disable printing?